spec/jobs/descriptive_metadata_import_job_spec.rb

```diff
@@ -100,6 +100,33 @@
       end
     end
 
+    context 'when index validation fails' do
```

Comment:
Seems like (1) DSC should be mocked and (2) that the error is written to the log should be tested.